### ISSUE TITLE
Problems of reproducing the transfering results with SimSiam/MoCo-v2 on COCO

### ISSUE THREAD

**Issue by thss15fyt** [facebookresearch/simsiam#26]
Hi, thanks for your contribution to this interesting work!

I am trying to reproduce the transferring results on COCO, based on Detectron2 and MoCo codebase. I reached the 38.5 AP_box and 33.7 AP_mask of MoCO-v2, which leaves a gap compared with the reported 39.2 AP_box and 34.3 AP_mask. I have noticed that SimSiam searched the learning rate for all the methods. Thus, could you please let me know the best configurations to reproduce the reported results on COCO? Thanks!

**Comment by endernewton** [comment]
I think it is listed in the paper already (with 10x the learning rate and weight decay changes).
